src/main/java/teammates/ui/webapi/GetHasResponsesAction.java

```diff
@@ -87,17 +109,33 @@ JsonResult execute() {
             return new JsonResult(new HasResponsesData(hasResponses));
         } else {
```

Comment:
this else is unnecessary - we already `return` above. This is not your change but please help to clean this up as well (since we are on this page)

Reply:
Got it! I do agree with some cleanup here!

Reply:
After some tinkering, removal of the `else` clause results in an un-bypassable lint failure from the nested `if` on top from pmdMain. Have to extract the instructor logic away into an individual method to avoid this, which should be more readable in a sense.